GooglePlayInstant/Editor/QuickDeploy/HttpRequestHelper.cs

```diff
@@ -0,0 +1,141 @@
+        public static WWW SendHttpPostRequest(string endpoint, Dictionary<string, string> postParams,
+            Dictionary<string, string> postHeaders)
+        {
+            var form = GetWwwForm(postParams);
+            return SendHttpPostRequest(endpoint, postParams != null ? form.data : null,
+                GetCombinedDictionary(form.headers, postHeaders));
+        }
+
+        /// <summary>
+        /// Returns a WWWform containing the body params to use for a POST request.
+        /// </summary>
+        internal static WWWForm GetWwwForm(Dictionary<string, string> postParams)
+        {
+            var form = new WWWForm();
+            if (postParams != null)
+            {
+                foreach (var pair in postParams)
+                {
+                    form.AddField(pair.Key, pair.Value);
+                }
+            }
+
+            return form;
+        }
+
+        /// <summary>
+        /// Sends a general GET request to the specified endpoint along with specified parameters and headers.
+        /// </summary>
+        /// <param name="endpoint">The endpoint where the GET request should be sent. Must have no query params.</param>
+        /// <param name="getParams">A collection of key-value pairs to be attached to the endpoint as GET
+        /// parameters.</param>
+        /// <param name="getHeaders">A collection of key-value pairs to be added to the request headers.</param>
+        /// <returns>A reference to the WWW instance representing the request.</returns>
+        public static WWW SendHttpGetRequest(string endpoint, Dictionary<string, string> getParams,
+            Dictionary<string, string> getHeaders)
+        {
+            return new WWW(GetEndpointWithGetParams(endpoint, getParams), null,
+                GetCombinedDictionary(new WWWForm().headers, getHeaders));
+        }
+
+
+        /// <summary>
+        /// Adds GET params to endpoint and returns result.
+        /// </summary>
+        /// <remarks>Assumes endpoint does not have any queries attached to it.</remarks>
+        internal static string GetEndpointWithGetParams(string endpoint, Dictionary<string, string> getParams)
+        {
+            var uriBuilder = new UriBuilder(endpoint);
+            if (getParams != null)
+            {
+                uriBuilder.Query = string.Join("&",
+                    getParams.Select(kvp => string.Format("{0}={1}", WWW.EscapeURL(kvp.Key), WWW.EscapeURL(kvp.Value)))
+                        .ToArray());
+            }
+
+            return uriBuilder.ToString();
+        }
+
+        /// <summary>
+        /// Combines two dictionaries into a single dictionary. Values in the second argument override values of the
+        /// first argument for every key that is present in both dictionaries.
+        /// </summary>
+        internal static Dictionary<string, string> GetCombinedDictionary(Dictionary<string, string> firstDict,
```

Comment:
Change firstDict to first or firstDictionary. Same below, replace all abbreviations for "Dict".

Reply:
Resolved.